<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>406</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Heatmap Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox_correlation_type">
      <item>
       <property name="text">
        <string>Pearson</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spearman</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kendall</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Tamanho</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLineEdit" name="input_title"/>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBox_ID_firstcolumn">
      <property name="text">
       <string>Primeira coluna é ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tipo de correlação</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QCheckBox" name="checkBox_plotar_correalacao">
      <property name="text">
       <string>Plotar o valor da correlação</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QComboBox" name="comboBox_tamanho">
      <item>
       <property name="text">
        <string>Pequeno</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Médio</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grande</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QPushButton" name="btn_instructions">
      <property name="text">
       <string>Instruções</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="btn_open">
      <property name="text">
       <string>Abrir planilha</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Cores</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QComboBox" name="comboBox_cor">
      <item>
       <property name="text">
        <string>Azul</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Verde</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vermelho</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="btn_preview">
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="btn_salvar">
      <property name="text">
       <string>Salvar Heatmap</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>365</width>
         <height>146</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_status">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: white</string>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
